plugins/modules/at.py

```diff
@@ -21,21 +21,15 @@
     description:
      - A command to be executed in the future.
     type: str
-  script_file:
```

Comment:
I don't see the new `chdir` option here.
May I suggest that you don't change the order of the options here as it causes a noisy git-diff

Reply:
still in WIP, will work on that this week-end.

Reply:
Thanks, was just taking a quick skim over the open PRs to see if I could help.

Reply:
>  May I suggest that you don't change the order of the options here as it causes a noisy git-diff

Ok will change that